org.qutebrowser.qutebrowser.yml

```diff
@@ -0,0 +1,40 @@
+---
+app-id: org.qutebrowser.qutebrowser
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qutebrowser
+rename-desktop-file: qutebrowser.desktop
+rename-icon: qutebrowser
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+cleanup:
+- "/lib/debug"
+modules:
+- pygments.yml
+- pypeg2.yml
+- markupsafe.yml
+- jinja2.yml
+- attrs.yml
+- pyyaml.yml
+- sip.yml
+- pyqt5.yml
+- pyqt5-webengine.yml
+
+- name: qutebrowser
+  buildsystem: simple
+  build-commands:
+  - pip3 install --no-index --no-compile --prefix=/app .
```

Comment:
Why use pip over here? The [build process](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/qutebrowser) is pretty straightforward.